docs/documentation/overview.md

```diff
@@ -106,15 +106,16 @@ Restart `ng serve` if you're running it, and Bootstrap 4 should be working on
 your app.
 
 ### Additional Commands
-* [ng new](new)
-* [ng init](init)
-* [ng serve](serve)
-* [ng generate](generate)
-* [ng test](test)
-* [ng e2e](e2e)
-* [ng build](build)
-* [ng get/ng set](config)
-* [ng docs](docs)
+* [ng new](new.md)
```

Comment:
Since these documents are going to end up on the WIKI, this change is a mistake (and all the ones below). We do not want `.md` extensions.